### PR TITLE
Changed the chapter icon

### LINKED ISSUE #8409: Swap the chapter and queue icon
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I'm aware that this is a request for NewPipe itself and that requests for adding a new service need to be made at [NewPipeExtractor](https://github.com/TeamNewPipe/NewPipeExtractor/issues).
- [X] I have taken the time to fill in all the required details. I understand that the feature request will be dismissed otherwise.
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Feature description

As title states swap the two icons, or change to new icon
Screen Shot 2022-05-17 at 12 19 24 PM
s

### Why do you want this feature?

To clear up any confusion, I still get confused between the two. The queue icon should be the numbered one as that makes more sense. 

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Marius1501**
#### What is it?
- [ ] Bugfix (user facing)
- [x] Feature (user facing)
- [ ] Codebase improvement (dev facing)
- [ ] Meta improvement to the project (dev facing)

#### Description of the changes in your PR

- Changed the chapter icon because it was too similiar to the queue icon

#### Before/After Screenshots/Screen Record

Before: 
grafik

After: 
grafik

#### Fixes the following issue(s)

- Fixes #8409

#### Due diligence
- [x] I read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md).

### CODE REVIEW

File: app/src/main/res/layout/player.xml [hunk 1]
```diff
@@ -216,7 +216,7 @@
                         android:paddingEnd="6dp"
                         android:paddingBottom="3dp"
                         android:scaleType="fitCenter"
-                        android:src="@drawable/ic_format_list_numbered"
```
Comment (Stypox) [inline app/src/main/res/layout/player.xml:219]: format_list_numbered doesn't seem to be used anymore now. Please remove it

Comment (mahendranv) [reply app/src/main/res/layout/player.xml:219]: It has been removed now

**Comment by Stypox** [comment]
For some reason it looks a bit off... Can you check if the proper XML tags have been used to make the icon centered?

File: app/src/main/res/layout/player.xml [hunk 2]
```diff
@@ -212,11 +212,10 @@
                         android:clickable="true"
                         android:focusable="true"
                         android:paddingStart="6dp"
-                        android:paddingTop="5dp"
```
Comment (Stypox) [inline app/src/main/res/layout/player.xml:215]: Now it's too high... Just center it with 3dp of paddingTop

**Review by Stypox — CHANGES_REQUESTED** [review]

**Review by Stypox — APPROVED** [review]
Thank you! Looks good to me

**Comment by enessgokalp** [comment]
Please get the old one. It's more beautiful than the new one

**Comment by Stypox** [comment]
Nope, that won't happen

**Pull request merged**